Sources/ConcordiumSwiftSdk/NodeClient.swift

```diff
@@ -3,11 +3,23 @@ import GRPC
 import NIOCore
 import NIOPosix
 
-public class Client {
-    let grpc: Concordium_V2_QueriesNIOClient
+public protocol NodeClient {
```

Comment:
Maybe NodeClient is still a bit too generic. Let's say you are doing a "unified wallet" that implements several wallet SDKs, then this one doesn't say clearly that it is the Concordium one. So I would suggest something like `ConcordiumClient` instead